### PR TITLE
Fixing a config_file test that suddenly broke..

### PR DESCRIPTION
**Pull request by ejoerns**
This collects all changes that needed to be done to reveal and fix the issue why in my setup the test `/config-file/system-serial` suddenly painted an annoying red in the green meadow of successful test runs.

### CODE REVIEW

File: src/context.c [hunk 1]
```diff
@@ -566,20 +566,12 @@ const RaucContext *r_context(void)
...
-		context->keyringdirectory = NULL;
-		context->signing_keyringpath = NULL;
-		context->mksquashfs_args = NULL;
-		context->casync_args = NULL;
+		g_clear_pointer(&context->certpath, g_free);
+		g_clear_pointer(&context->keypath, g_free);
+		g_clear_pointer(&context->keyringpath, g_free);
+		g_clear_pointer(&context->keyringdirectory, g_free);
+		g_clear_pointer(&context->signing_keyringpath, g_free);
+		g_clear_pointer(&context->mksquashfs_args, g_free);
```
Comment (jluebbe) [inline src/context.c]: You lost casync_args here, I'll push a fixup myself.

File: src/context.c [hunk 2]
```diff
@@ -563,6 +563,12 @@ const RaucContext *r_context(void)
 	return context;
 }
 
+void r_context_install_info_free(RContextInstallationInfo *info)
+{
+	/* contains only reference to existing bundle instance */
+	info->mounted_bundle = NULL;
+}
```
Comment (jluebbe) [inline src/context.c]: Don't we need to free the struct itself here?

Comment (ejoerns) [reply src/context.c]: Oh, yes. seems I missed that.

File: src/context.c (hunk 1 above)
Comment (jluebbe) [reply src/context.c]: This is now fixed.

File: src/context.c (hunk 2 above)
Comment (jluebbe) [reply src/context.c]: OK, i'll add that as well.

**Pull request merged**